tests/src/test/scala/cats/tests/NonEmptyMapSuite.scala

```diff
@@ -201,4 +201,10 @@ class NonEmptyMapSuite extends CatsSuite {
       nem.length should ===(nem.toSortedMap.size)
     }
   }
+
+  test("NonEmptyMap#toNonEmptyList is consistent with Map#toList and creating NonEmptyList from it"){
+    forAll{ nem: NonEmptyMap[String, Int] =>
+      nem.toNonEmptyList should ===(NonEmptyList.fromListUnsafe(nem.toSortedMap.toList))
```

Comment:
Unfortunately the `toNonEmptyList` of `NonEmptyMap` collides with `toNonEmptyList` of `Reducible`, so the `nem.toNonEmptyList` here seems to return a `NonEmptyList[Int]` instead of the `NonEmptyList[(String, Int)]` that we want. 
Maybe we should rename the method to `toNel` instead? :)

Reply:
Worked, thanks a lot!